cmsplugin_offices_ai/models/personnel.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -:-
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import gettext_lazy as _
+from filer.fields.image import FilerImageField
+
+from cmsplugin_offices_ai.models import Department, Office
+
+
+class PersonQuerySet(models.QuerySet):
+
+    def in_order(self):
+        """
+        Orders queryset by `ordering` field value
+        """
+        return self.order_by('ordering')
+
+    def public(self):
+        """
+        Exclude personnel that are not supposed to be shown in lists from queryset.
+        """
+        return self.filter(show_in_list=True).prefetch_related('person_contact_information')
+
+
+@python_2_unicode_compatible
+class Person(models.Model):
+    name = models.CharField(
+        verbose_name=_('name'),
+        max_length=128,
+        blank=False,
+    )
+    title = models.CharField(
+        verbose_name=_('title'),
+        max_length=128,
+        blank=True,
+    )
+    description = models.TextField(
+        verbose_name=_('description'),
+        max_length=512,
+        blank=True,
+    )
+    image = FilerImageField(
+        verbose_name=_('image'),
+        related_name='+',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+
+    office_override = models.ForeignKey(
```

Comment:
Person might belong to a department that belongs to different office the person actually works at. I figured this might be somewhat common special case in real world.